### PR TITLE
install: drop support matrix and update instructions

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind ci
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design
/kind documentation
> /kind failing-test
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
/lgtm

File: install.md [hunk 1]
```diff
@@ -39,19 +39,32 @@ It is assumed you are running a Linux machine.
...
 
-|      | Fedora 31+ | openSUSE | CentOS_8 | CentOS_8_Stream | CentOS_7 | Debian_Unstable | Debian_Testing | Debian 10 | Rasbian_10 | xUbuntu_20.04 | xUbuntu_19.10 | xUbuntu_19.04 | xUbuntu_18.04 |
-| ---- | ---------- | -------- | -------- | --------------- | -------- | --------------- | -------------- | --------- | ---------- | ------------- | ------------- | ------------- | ------------- |
-| 1.18 | ✓          | ✓        | ✓        | ✓               |          | ✓               | ✓              |           |            | ✓             |               |               |               |
-| 1.17 | ✓          | ✓        | ✓        | ✓               | ✓        | ✓               | ✓              | ✓         | ✓          | ✓             | ✓             | ✓             | ✓             |
-| 1.16 | ✓          | ✓        | ✓        | ✓               | ✓        | ✓               | ✓              | ✓         | ✓          | ✓             | ✓             | ✓             | ✓             |
+CRI-O follows the [Kubernetes support cycle](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions) of three minor releases.
```
Comment (saschagrunert) [inline install.md:43]: We're about to move that doc to k/release/versioning.md. Ref: https://github.com/kubernetes/community/pull/5632#pullrequestreview-612120551

**Comment by TomSweeneyRedHat** [comment]
LGTM
IDK if you want to address @saschagrunert latest comment now or in a follow up.

**Pull request merged**